cmake/IgnPython.cmake

```diff
@@ -12,14 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Copied from ament/ament_cmake: ament_cmake/ament_cmake_core/cmake/core/python.cmake
+if(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.19")
+  set(PYTHON_VERSION "" CACHE STRING
+    "Specify specific Python3 version to use ('major.minor' or 'versionMin...[<]versionMax')")
 
-set(PYTHON_VERSION "" CACHE STRING
-  "Specify specific Python version to use ('major.minor' or 'major')")
+  find_package(Python3 ${PYTHON_VERSION} QUIET)
+elseif(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.12")
+  # no support for finding specific versions
+  find_package(Python3 QUIET)
```

Comment:
I was thinking on warning the user if  `PYTHON_VERSION` is being set but CMake is already doing it for using of CMake lower than 3.19 and greater than 3.12 (like Focal).